Assets/MirageXR/Services/IBM-Watson/DialogueService.cs

```diff
@@ -122,7 +123,7 @@ public async Task SendMessageToAssistantAsync(string text)
                         return;
                     }
                     
-                    AppLog.Log("[DialogueService] starting to parse", LogLevel.INFO);
+                    AppLog.Log("[DialogueService] starting to parse", LogLevel.LogLevel.INFO);
```

Comment:
isn't this now mixing two styles in one line? Should it be LogLevel.AppLog.Log?

Reply:
forgot to remove. `Photon.LogLevel` will conflict with `i5.LogLevel` since LogLevel from i5 does not have a namespace. @BenediktHensen , can you fix this? I wrote to you about this.


![image](https://github.com/user-attachments/assets/9afce904-06a2-46eb-a057-873ec2602841)
![image](https://github.com/user-attachments/assets/37aabcb9-3b57-47f5-a4c1-acb1052d5654)